fatiando/mesher.py

```diff
@@ -1720,3 +1720,433 @@ def keep(cell):
             return True
         return False
     return [c for c in cells if keep(c)]
+
+
+class TriaxialEllipsoid (GeometricElement):
+
```

Comment:
Watch out for these extra white spaces before the `(` and the extra newline between a function/class and the docstring. They appear in a lot of places below. Just remove them.